### ISSUE TITLE
运行期异常

### ISSUE THREAD

**Issue by shixinglingguihua** [wangzhegeek/EGES#1]
运行环境
python 3.7.3
tensorflow == 1.14.0
joblib
networkx

运行步骤：
python  data_process.py
pyhon   EGES.py

报错信息：

Errors may have originated from an input operation.
Input Source operations connected to node embedding_lookup_2:
 inputs_shop (defined at F:/PycharmProjects/EGES-master/EGES.py:70)	
 Variable_2/read (defined at F:/PycharmProjects/EGES-master/EGES.py:79)

Original stack trace for 'embedding_lookup_2':
  File "F:/PycharmProjects/EGES-master/EGES.py", line 80, in <module>
    shop_embed = tf.nn.embedding_lookup(shop_embedding, inputs_shop)
  File "C:\ProgramData\Anaconda3\lib\site-packages\tensorflow\python\ops\embedding_ops.py", line 315, in embedding_lookup
    transform_fn=None)
  File "C:\ProgramData\Anaconda3\lib\site-packages\tensorflow\python\ops\embedding_ops.py", line 133, in _embedding_lookup_and_transform
    array_ops.gather(params[0], ids, name=name), ids, max_norm)
  File "C:\ProgramData\Anaconda3\lib\site-packages\tensorflow\python\util\dispatch.py", line 180, in wrapper
    return target(*args, **kwargs)
  File "C:\ProgramData\Anaconda3\lib\site-packages\tensorflow\python\ops\array_ops.py", line 3475, in gather
    return gen_array_ops.gather_v2(params, indices, axis, name=name)
  File "C:\ProgramData\Anaconda3\lib\site-packages\tensorflow\python\ops\gen_array_ops.py", line 4097, in gather_v2
    batch_dims=batch_dims, name=name)
  File "C:\ProgramData\Anaconda3\lib\site-packages\tensorflow\python\framework\op_def_library.py", line 788, in _apply_op_helper
    op_def=op_def)
  File "C:\ProgramData\Anaconda3\lib\site-packages\tensorflow\python\util\deprecation.py", line 507, in new_func
    return func(*args, **kwargs)
  File "C:\ProgramData\Anaconda3\lib\site-packages\tensorflow\python\framework\ops.py", line 3616, in create_op
    op_def=op_def)
  File "C:\ProgramData\Anaconda3\lib\site-packages\tensorflow\python\framework\ops.py", line 2005, in __init__
    self._traceback = tf_stack.extract_stack()

**Comment by wangzhegeek** [comment]
这个主要是id不是从0开始的，在lookup的时候会出现index错误，已更新
